frontend/packages/console-shared/src/components/editor/yaml-editor-utils.ts

```diff
@@ -7,8 +7,8 @@ import { getLanguageService, TextDocument } from 'yaml-language-server';
 import { openAPItoJSONSchema } from '@console/internal/module/k8s/openapi-to-json-schema';
 import { getStoredSwagger } from '@console/internal/module/k8s/swagger';
 import {
-  global_BackgroundColor_100 as lineNumberActiveForeground,
```

Comment:
Hm, I would expect `global_BackgroundColor_100` to still be white based on https://patternfly-pr-3139.surge.sh/documentation/overview/global-css-variables. Is this a Patternfly bug?

@jcaianirh @dtaylor113

Reply:
I'm seeing it defined in: `frontend/node_modules/@patternfly/react-tokens/dist/js/global_BackgroundColor_100.d.ts`
```
export const global_BackgroundColor_100: {
  "name": "--pf-global--BackgroundColor--100",
  "value": "#151515",
  "var": "var(--pf-global--BackgroundColor--100)"
};
export default global_BackgroundColor_100
```
#151515 is a black color

Reply:
In `https://patternfly-pr-3139.surge.sh/documentation/overview/global-css-variables#global-css-variables` it is defined as `--pf-global--BackgroundColor--dark-100`.  I do see a 'light' theme, ie:  `--pf-global--BackgroundColor--light-100`...`...200` etc..

Reply:
I'm still confused because `--pf-global--BackgroundColor--100` is defined as `#fff` on https://patternfly-pr-3139.surge.sh/documentation/overview/global-css-variables

<img width="1029" alt="Screen Shot 2020-07-20 at 9 56 39 AM" src="https://user-images.githubusercontent.com/1167259/87945914-61403580-ca6f-11ea-891c-89d5a666701d.png">

Reply:
[Core is the source of truth](https://github.com/patternfly/patternfly/blob/master/src/patternfly/sass-utilities/scss-variables.scss#L26) for this which defines `$pf-global--BackgroundColor--100: $pf-color-white !default;` where `$pf-color-white: #fff !default;`. Our documentation site shows the correct value.


Our react-token `global_BackgroundColor_100.js` is wrong (and has been wrong) for reasons @jschuler can explain. We recommend using `@patternfly/react-tokens/dist/js/patternfly_variables.js` which has the correct value `#fff`.

Reply:
Unfortunately some of the generated tokens have wrong values, i've filed a bug here https://github.com/patternfly/patternfly-react/issues/4583.
The reason for the bug is that variables are sometimes redefined in components, and the token generation script wrongly keeps the last value in finds in the patternfly scss files. 
The `patternfly_variables` file has the correct values but it contains all the global vars so is larger.

Reply:
I'd be OK switching to `BackgroundColor--light-100` as well. I just wanted to check whether there was a PF bug here so we can file it upstream. Thanks @redallen and @jschuler !